WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItem.swift

```diff
@@ -158,12 +161,20 @@ struct WooShippingCustomsItem: View {
         .sheet(isPresented: $isShowingCountries, content: {
             NavigationStack {
                 SingleSelectionList(title: Localization.originCountryTitle,
-                                    items: viewModel.allCountries,
+                                    items: viewModel.countries,
                                     contentKeyPath: \.name,
                                     selected: $viewModel.originCountry)
```

Comment:
What do you think of using the `FilterListSelector` view instead of `SingleSelectionList`, so the merchant can search for the country they want to select?

Reply:
That's a good idea! As using `FilterListSelector` is not so straightforward in this case, I created an [issue](https://github.com/woocommerce/woocommerce-ios/issues/14813) so we can address this later on.